eotsmanager/proto/eotsmanager.proto

```diff
@@ -73,6 +69,9 @@ message CreateRandomnessPairListRequest {
   uint64 start_height = 3;
   // num is the number of randomness pair list
   uint32 num = 4;
+  // interval is the optional height interval between consecutive randomness pairs
+  // If not set or 0, consecutive heights are used
+  optional uint64 interval = 5;
```

Comment:
What do you think we have this as mandatory and require the client to use `1` explicitly for consecutive heights? 

Right now `interval == nil` and `interval == &1` have the same meaning, and type `*uint64` is a bit weird in Go